src/request/components/RequestDetailPanelRequest.tsx

```diff
@@ -79,4 +82,12 @@ export class Request extends React.Component<IRequestProps, {}> {
             </div>
         );
     }
+
+    private renderRaw(body: string) {
+        return (
+            <div className='tab-request-raw'>
```

Comment:
TypeScript leans towards single quotes `'` to delimit strings and we have linting rules to that effect (at least in other repos).  Unless, you mean something entirely different.